R/expect-length.R

```diff
@@ -1,6 +1,7 @@
 #' Does code return a vector with the specified length?
 #'
-#' @seealso [expect_vector()] to make assertions about the "size" of a vector
+#' @seealso [expect_vector()] to make assertions about the "size" of a vector,
```

Comment:
My docker image doesn't have documentation files so I haven't managed to `roxygenize()` yet...